### DISCUSSION TITLE
Where are the logs when using a Docker install?

### DISCUSSION THREAD

**Discussion by camfam** [firefly-iii/firefly-iii#5454] [Firefly III] [answered]
Trying to identify why the email says sent, but it is not received.  Went to check the logs but I can't find them?

Where are the logs on a docker install?  (I feel like an idiot)

**Comment by JC5** [comment]
They're being printed to stdout, so you use `docker logs <container>`

If you wish to read the files, you have to mount a volume to the directory `/var/www/html/storage/logs`, for example (command line):

```
-v /home/james/temp-logs:/var/www/html/storage/logs
```

**Comment by camfam** [comment, marked as answer]
Excellent , James!  

[2021-12-30T05:46:13.213029+00:00] local.INFO: AUDIT: User sends test message. (192.168.1.1 ***@***.***) -> POST:https://subd.domain.com/admin/test-message) [] [

… is what I’m seeing in the stdout stream.  I’m guessing that I need to enable debug or verbose logging?  If so, how?


On Dec 30, 2021, at 1:04 AM, James Cole ***@***.***> wrote:


They're being printed to stdout, so you use docker logs <container>

If you wish to read the files, you have to mount a volume to the directory /var/www/html/storage/logs, for example (command line):

-v /home/james/temp-logs:/var/www/html/storage/logs
—
Reply to this email directly, view it on GitHub <https://github.com/firefly-iii/firefly-iii/discussions/5454#discussioncomment-1886013>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AM7DZZ2BXRMIB72ZEMWHNETUTPY7RANCNFSM5K7EO2QA>.
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by JC5** [reply]
https://docs.firefly-iii.org/firefly-iii/faq/other/#how-do-i-enable-debug-mode

**Comment by camfam** [comment]
Perfect.  No errors in the logs.  Unfortunately, no email was received either, so the problem lies elsewhere.  Probably the mail relay or the email host itself.

Thank you, James!  You've helped me narrow down which rocks to turn over.
